src/r_segs.c

```diff
@@ -566,6 +567,42 @@ void R_StoreWallRange(const int start, const int stop)
   ds_p->curline = curline;
   rw_stopx = stop+1;
 
+  { // killough 1/6/98, 2/1/98: remove limit on openings
+    static ptrdiff_t maxopenings;
+
+    ptrdiff_t pos = lastopening - openings;
+    ptrdiff_t need = (rw_stopx - start) * sizeof(*lastopening) + pos;
```

Comment:
Sure we need `sizeof()` in pointer arithmetics?

Reply:
Yes, `rw_stopx` and `start` are indexes and we need address in array.